.github/workflows/sql-cli-test-and-build-workflow.yml

```diff
@@ -28,26 +37,21 @@ jobs:
           pip install -r requirements-dev.txt
           pip install setuptools wheel
       
-      #TODO: will setup CI for IT once we have OpenSearch and sql plugin release. Not it only runs UT
-      # It can also be refactored by launching OpenSearch instance with plugin installed from gradle.
-      
-      # - name: Set up ES and install SQL plugin
-      #   run: |
-      #     sudo add-apt-repository ppa:openjdk-r/ppa
-      #     sudo apt update
-      #     sudo apt install openjdk-14-jdk
-      #     sudo apt install unzip
-      #     wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.10.0-amd64.deb
-      #     sudo dpkg -i elasticsearch-oss-7.10.0-amd64.deb
-      #     sudo /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opensearch-sql/opensearch_sql-1.12.0.0.zip
-      #     sudo systemctl start elasticsearch.service
+        # tests are designed to run against http://localhost:9200, so we have to disable/remove security plugin
+      - name: Download and run OpenSearch
+        run: |
+          docker run -p 9200:9200 -e "discovery.type=single-node" -e "DISABLE_SECURITY_PLUGIN=true" --name test -d opensearchproject/opensearch:${{ matrix.opensearch-version }}
```

Comment:
it may not work. e.g, currently, SQL plugin depend on OpenSearch 2.4. But OpenSearch 2.4 Docker Images is not released yet.

Reply:
Currently, there are only `2.2.1` and `latest`. `latest` grabs the latest released version.
I can create another sub-job to download and build OpenSearch. So tests would run on the `current` version, e.g. 2.4 for now.

Reply:
Might be worthwhile the next time someone has a chance. 
Another side-effect, is that we would have an artefact to download for our own local development usage... instead of having to download and setup the latest OpenSearch.